servers/src/main/java/tachyon/master/Master.java

```diff
@@ -20,9 +20,9 @@
 import org.apache.thrift.TProcessor;
 
 import tachyon.master.journal.JournalCheckpointStreamable;
-import tachyon.master.journal.JournalEntry;
 import tachyon.master.journal.JournalInputStream;
 import tachyon.master.journal.ReadWriteJournal;
+import tachyon.proto.JournalEntryProtos.JournalEntry;
```

Comment:
if that's the case, I suggest we call the file `journal.proto` do decrease redundancy